fetools-app/src/pages/Tables.jsx

```diff
@@ -64,45 +70,43 @@ const Table = () => {
         title="Code Snippets"
         icon="integration_instructions"
       >
-        <TabSwitcher title="" buttons={['css', 'tailwind']}>
-          <CodeBlock title="" code={''} lang="css" />
+        {/* <CodeGeneratorTable toastState={toastState} tableConfig={tableConfig}  /> */}
+
+        <TabSwitcher buttons={["HTML & CSS", "Tailwind"]}>
+          <CodeBlock
+            toastState={toastState}
+            code={generateTableCode(tableConfig)}
+            lang="html" 
+          />
```

Comment:
We want the block title there for consistency and clarity.

```suggestion
          <CodeBlock
            title="HTML & CSS"
            toastState={toastState}
            code={generateTableCode(tableConfig)}
            lang="html" 
          />
```